app/controllers/DetermineTaxIdController.scala

```diff
@@ -69,6 +73,7 @@ class DetermineTaxIdController @Inject() (
         case TaxRegime.Epaye => Redirect(routes.NotEnrolledController.notEnrolled)
         case TaxRegime.Vat   => Redirect(routes.NotEnrolledController.notVatRegistered)
         case TaxRegime.Sa    => Redirect(routes.NotEnrolledController.notSaEnrolled)
+        case TaxRegime.Sia   => Redirect(routes.NotEnrolledController.notSaEnrolled)
```

Comment:
perhaps we should just throw an exception here for now instead of going to a page for a different tax regime. I'm not sure if we're gonna have a dedicated page for this scenario actually, so it'd be the least amount of work in the future to throw here if we end up not having one

Reply:
oh wait a sec, looks like you've got a `siaNoNino` endpoint already - I guess this should point to that endpoint instead. It looks like we are having a kickout page having looked at another ticket

Reply:
yeah, I just forgot to change this to point to the no nino endpoint, mb